### PR TITLE
Adds labels to batch buttons

### PR DESCRIPTION
**Pull request by stuartromanek**
Feedback says the icon-only look is hard to parse. this PR adds labels to the buttons.

## Summary

- Adds labels to batch operation buttons
- makes UI inside manager toolbars a bit smaller
- don't render the toolbar at all if there are no pieces to display

## What are the specific steps to test this change?

Should look like
image

1. link to demo, npm run dev
2. open the User manager to see the label'd batch buttons
3. open any other manager with no pieces to see that the toolbar doesn't appear when there are no pieces to display

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [ ] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue [hunk 1]
```diff
@@ -64,6 +65,7 @@
         :status="searchField.status" :value="searchField.value"
         :modifiers="['small']"
         ref="search"
+        class="apos-manager-toolbar__search"
```
Comment (haroun) [inline modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue:68]: bit confused by this change to be honest
Are we using this new class?

Comment (stuartromanek) [reply modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue:68]: seems not, will remove

**Review by haroun — APPROVED** [review]
